recipes/prinseq-plus-plus/build.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set +ex
 
+export CXXFLAGS="$CXXFLAGS -I$PREFIX/include -L$PREFIX/lib"
```

Comment:
If you do it that way, you ignore other `LDFLAGS` we have set in the build setup, i.e., `-rpath` flags.
I suggest to adjust the Makefile to use `$(LDFLAGS) $(LIBS)` when linking the binaries.

Reply:
Perfect, thanks for changing it! I'm grateful for those tiny bits and peaces we don't have to maintain in the recipes themselves.!

Reply:
Terrific idea. Done and all checks passed! Thanks